Helper/Config.php

```diff
@@ -80,6 +99,32 @@ public function getNotificationsIpHmacCheck($storeId = null)
         );
     }
 
+    /**
+     * Retrieve key for notifications_hmac_key
+     *
+     * @param int $storeId
+     * @return string
+     */
+    public function getNotificationsHmacKey($storeId = null)
+    {
+        $key = "";
+        if ($this->adyenHelper->isDemoMode($storeId)) {
```

Comment:
Since isDemoMode() is a config function and this is the Config helper we can move it to this class now that we need it (flag/bool kind like getNotificationsCanCancel()). We want to ease the dependencies on the Data.php helper as it is getting bigger and will probably be slimmed down in the future

Reply:
Good idea. I removed the Data.php dependency and add an idDemoMode method